PA193_mnemonic_Slytherin/mnemonic.py

```diff
@@ -25,7 +25,76 @@
 SEED_LEN = 64
 
 
-class Mnemonic:
+class Seed(bytes):
+    """Class for seed representation.
+    """
+    
+    def __init__(self, seed: bytes):
+        """Check whether provided bytes represent a valid seed.
+        :raises ValueError: on invalid parameters
+        """
+        if not isinstance(seed, bytes) or len(seed) != SEED_LEN:
+            raise ValueError('Cannot instantiate seed')
+        self.__repr__ = seed
```

Comment:
This is not how `repr` works. It should just call superclass constructor with bytes.

Reply:
bytes is an immutable type, do you know of a way how to do this?

Reply:
please check it out if its acceptable now